terraform/modules/forest_node/user-data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Create a login user
+adduser --home "/home/${NEW_USER}" --disabled-password --gecos "" "${NEW_USER}"
+
+# Create .ssh directory for created user
+mkdir --parents -- "/home/${NEW_USER}/.ssh"
+chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh"
+chmod 0700 "/home/${NEW_USER}/.ssh"
+
+# Check if authorized_keys file exists and copy it from root to new user
+if [ -f "/root/.ssh/authorized_keys" ]; then
+  cp /root/.ssh/authorized_keys "/home/${NEW_USER}/.ssh/authorized_keys"
+  chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh/authorized_keys"
+  chmod 0600 "/home/${NEW_USER}/.ssh/authorized_keys"
+fi
+
+# Allow created user to log in
+echo "AllowUsers ${NEW_USER}" >> /etc/ssh/sshd_config
+
+# Restart SSHD
+systemctl restart sshd
+
+# Add user to sudoers file. We pipe to visudo to check syntax
+usermod --append --groups sudo "${NEW_USER}"
+
+# Add new user to "docker" group
+usermod --append --groups docker "${NEW_USER}"
+
```

Comment:
```suggestion
# We want the user to be able to run a docker container, and to have general admin rights
usermod --append --groups sudo,docker "${NEW_USER}"

```

Reply:
I'm not really able to audit the security of these changes because I don't have an idea of _why_ this file exists, what the VM is for, etc. The README doesn't really tell me - where can future readers find this information?